src/utils/session-helper.js

```diff
@@ -272,6 +274,48 @@ const createFileChecksum = async (fileObject) => {
   return sha256(text);
 };
 
+/**
+ * Helper Function that returns Driver's License ttl file based off of image passed
+ *
+ * @memberof utils
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * @param {string} documentUrl - url of uploaded document or resource
+ */
+
+const createDriversLicenseTtlFile = async (fileObject, documentUrl, checksum) => {
+  const dlData = await getDriversLicenseData(fileObject.file);
+  return buildThing(createThing({ name: 'document' }))
+    .addDatetime(RDF_PREDICATES.uploadDate, new Date())
+    .addStringNoLocale(RDF_PREDICATES.additionalType, dlData.DCA)
+    .addStringNoLocale(RDF_PREDICATES.conditionsOfAccess, dlData.DCB)
+    .addStringNoLocale(RDF_PREDICATES.expires, `${formattedDate(dlData.DBA)}`)
```

Comment:
I think we should try to have these with Dates as .addDatetime instead of .addStringNoLocale. But having them as string is okay.

Reply:
Makes total sense. I'll push these changes as well. Thanks for catching these.

Reply:
On a second pass look at this. I'm getting a strange message back from solid server when I attempt to load the driver's license image when using the .addDate method instead of the .addStringNoLocale method. It basically says it can't retrieve the data [Object object]. Same as with the height. I've checked the types of the passed data against @inrupt's documentation, this should work as it still expects a string, but perhaps the @inrupt expects the format of the date to be different? That also could be the same as their .addInteger function though that one makes less sense as I tried to coerce those values to Number() and it still didn't work. I'm a bit at a loss.

I'll have to take a closer look at @inrupt's documentation (though it is sparse imho) to get a better understanding.

Reply:
That is certainly strange. At the very least, I was expecting .addInteger to work because it was one of the examples shown in their documentation: https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/read-write-data/#get-the-thing-to-modify

Might have to do a little testing on the side

Reply:
I think I found the issue. It's because .addDate requires you to pass in a Date object, but what we get from `formattedDate` is just the parsed string. To make sure that it works, we need to update the value using:

```
new Date(`${formattedDate(dlData.<3-letter-code>)}`)
```

Reply:
Nice find. I'll make those changes on my end and push if it works for me.

Reply:
Oh, do you mind providing a link to the docs where you found this? I'm just curious.

Reply:
For the Date Object specification here's the link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format

For .addDate, I've just doubled check the type .addDate requires for their parameters:

<img width="646" alt="Screen Shot 2023-05-18 at 2 38 08" src="https://github.com/codeforpdx/PASS/assets/14917816/1181f962-8358-4d04-9b27-64eb4c9e46ef">